### DISCUSSION TITLE
An native way to listen 'alpine:initializing'

### DISCUSSION THREAD

**Discussion by luizbills** [alpinejs/alpine#1448] [3. Feature ideas]
In browser we need to use this event `document.addEventListener('alpine:initializing', callback)` to load data components... Maybe we can make the life easier implementing a third argument in `Alpine.data` to indicate whether or not this component should be loaded in that event. 

```js
const dropdown = () => ({
  open: false,

  toggle() {
    this.open = ! this.open
  }
})

// `true` to load automatically in the event "alpine:initializing"
Alpine.data('dropdown', dropdown, true)

// `false` when using in a module/bundle (default value maybe?)
Alpine.data('dropdown', dropdown, false)
```

**Comment by luizbills** [comment]


**Reply by ekwoka** [reply]
This doesn't work, because you don't have Alpine. that's the issue.
